chia/full_node/full_node.py

```diff
@@ -185,10 +190,25 @@ def sql_trace_callback(req: str):
         self.log.info("Initializing blockchain from disk")
         start_time = time.time()
         reserved_cores = self.config.get("reserved_cores", 0)
+        multiprocessing_start_method = process_config_start_method(
+            method=self.config.get("multiprocessing_start_method"),
```

Comment:
There is no default value here, should we set one?

Reply:
The default here is `None`, albeit implicitly, just to make the config less picky and allow bare `multiprocessing_start_method:` to be valid.